src/tex/appendix.tex

```diff
@@ -447,8 +447,19 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.48 on \ChemBench corpus, which is only 10\% less accurate than the much larger \LlamaThreeOneFourZeroFiveBInstruct model.
+
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo or \GPTFourO > \GPTFour > \GPTThreeFiveTurboZeroT). Newer versions consistently outperform their predecessors, indicating improvements in scientific understanding of these model families.
```

Comment:
```suggestion
One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo or \GPTFourO > \GPTFour > \GPTThreeFiveTurboZeroT). Newer versions consistently outperform their predecessors.
```

not sure if we can generalize to scientific understanding. perhaps chemical understanding